android/libraries/rib-base/src/main/java/com/badoo/ribs/core/Router.kt

```diff
@@ -22,10 +23,12 @@ import com.badoo.ribs.core.routing.configuration.feature.BackStackFeature.Operat
 import com.badoo.ribs.core.routing.configuration.feature.BackStackFeature.Operation.Replace
 import com.badoo.ribs.core.routing.configuration.feature.ConfigurationFeature
 import com.badoo.ribs.core.routing.configuration.toCommands
+import com.badoo.ribs.core.routing.transition.handler.TransitionHandler
 import com.badoo.ribs.core.view.RibView
 
 abstract class Router<C : Parcelable, Permanent : C, Content : C, Overlay : C, V : RibView>(
     savedInstanceState: Bundle?,
+    private val transitionHandler: TransitionHandler<C>? = null,
```

Comment:
let's define it as a last parameter to not break unnamed elements
also not sure if we should include it to template directly
I see it as a bloat for many cases, as we once again optimize for maximal case instead of minimal.

Reply:
Moving to end: sure
Adding to template: plugin checkboxes will help with that